test/unit/mdc-slider/foundation-keyboard-events.test.js

```diff
@@ -0,0 +1,661 @@
+
+  td.when(mockAdapter.computeBoundingRect()).thenReturn({left: 0, width: 100});
+  foundation.init();
+  raf.flush();
+
+  foundation.setValue(50);
+  raf.flush();
+
+  rootHandlers.keydown(createFakeEvent({keyCode: 33}));
+  raf.flush();
+
+  assert.equal(foundation.getValue(), 54);
+  td.verify(mockAdapter.setThumbContainerStyleProperty(TRANSFORM_PROP, 'translateX(54px) translateX(-50%)'));
+  td.verify(mockAdapter.setTrackStyleProperty(TRANSFORM_PROP, 'scaleX(0.54)'));
+
+  raf.restore();
+});
+
+test('on page down keydown prevents default', () => {
+  const {foundation, mockAdapter, raf, rootHandlers} = setupTest();
+  const evt = createFakeEvent({key: 'PageDown'});
+
+  td.when(mockAdapter.computeBoundingRect()).thenReturn({left: 0, width: 100});
+  foundation.init();
+  raf.flush();
+
+  rootHandlers.keydown(evt);
+  raf.flush();
+
+  td.verify(evt.preventDefault());
+
+  raf.restore();
+});
+
+test('on page down keydown increases the slider by 4 when no step given', () => {
+  const {foundation, mockAdapter, raf, rootHandlers} = setupTest();
+
+  td.when(mockAdapter.computeBoundingRect()).thenReturn({left: 0, width: 100});
+  foundation.init();
+  raf.flush();
+
+  foundation.setValue(50);
+  raf.flush();
+
+  rootHandlers.keydown(createFakeEvent({key: 'PageDown'}));
+  raf.flush();
+
+  assert.equal(foundation.getValue(), 46);
+  td.verify(mockAdapter.setThumbContainerStyleProperty(TRANSFORM_PROP, 'translateX(46px) translateX(-50%)'));
+  td.verify(mockAdapter.setTrackStyleProperty(TRANSFORM_PROP, 'scaleX(0.46)'));
+
+  raf.restore();
+});
+
+test('on page down keydown increases the slider by 4 * the step value when a step is given', () => {
+  const {foundation, mockAdapter, raf, rootHandlers} = setupTest();
+
+  td.when(mockAdapter.computeBoundingRect()).thenReturn({left: 0, width: 100});
+  foundation.init();
+  raf.flush();
+
+  foundation.setValue(50);
+  foundation.setStep(5);
+  raf.flush();
+
+  rootHandlers.keydown(createFakeEvent({key: 'PageDown'}));
+  raf.flush();
+
+  assert.equal(foundation.getValue(), 30);
+  td.verify(mockAdapter.setThumbContainerStyleProperty(TRANSFORM_PROP, 'translateX(30px) translateX(-50%)'));
+  td.verify(mockAdapter.setTrackStyleProperty(TRANSFORM_PROP, 'scaleX(0.3)'));
+
+  raf.restore();
+});
+
+test('on page down keydown works with keycode', () => {
+  const {foundation, mockAdapter, raf, rootHandlers} = setupTest();
+
+  td.when(mockAdapter.computeBoundingRect()).thenReturn({left: 0, width: 100});
+  foundation.init();
+  raf.flush();
+
+  foundation.setValue(50);
+  raf.flush();
+
+  rootHandlers.keydown(createFakeEvent({keyCode: 34}));
+  raf.flush();
+
+  assert.equal(foundation.getValue(), 46);
+  td.verify(mockAdapter.setThumbContainerStyleProperty(TRANSFORM_PROP, 'translateX(46px) translateX(-50%)'));
+  td.verify(mockAdapter.setTrackStyleProperty(TRANSFORM_PROP, 'scaleX(0.46)'));
+
+  raf.restore();
+});
+
+test('on any other keydown does nothing', () => {
```

Comment:
nit: I mean... kind of. This really just tests if any other key is the Enter key.